README.rst

```diff
@@ -141,6 +141,11 @@ Successful completion.
 An error occurred.
 
 
+2
```

Comment:
I could not find the location of where this is implemented. Did you maybe forget to stage the changed file?

Reply:
This is done by argparser. That's not an implementation of me.

Reply:
:+1: